common/ethnicities/wc_ethnicities_orcish.txt

```diff
@@ -1,25 +1,16 @@
 ﻿###Hair colors
-@blonde_color_x_min = 0.2
```

Comment:
I get a crash when I open the Ears tab in the Ruler Designer. If you can't find a bug and need help, ping me. Also, fix errors first. It may be them.